README.md

```diff
@@ -147,13 +137,23 @@ The breakpoints are configured in `./src/scss/settings/_settings.breakpoints.scs
 ## File Generators
 To quickly add new parts to your app like components modules etc. the syntax to you can use is
 ```sh
-yo emakinacee-react:gen type name module
+react g <scaffold> <name> [module]
 ```
+
+Scaffold  | Usage
+---       | ---
+[Component](https://github.com/emakina-cee-oss/react-cli/wiki) | `react g component AwesomeComponent`
+[Module](https://github.com/emakina-cee-oss/react-cli/wiki)    | `react g module AwesomeModule`
+[Service](https://github.com/emakina-cee-oss/react-cli/wiki)   | `react g service AwesomeService`
+[Action](https://github.com/emakina-cee-oss/react-cli/wiki)    | `react g action awesomeAction SomeModule`
+[Factory](https://github.com/emakina-cee-oss/react-cli/wiki)   | `react g factory awesomeFactory SomeModule`
+[Compute](https://github.com/emakina-cee-oss/react-cli/wiki)   | `react g compute awesomeCompute SomeModule`
+
 See the following examples for more detail.
 
 ### Component
 ```sh
-yo emakinacee-react:gen component ComponentName
+react g component ComponentName
```

Comment:
This is a really good point. I was wondering about those upper case file names too when I started to look into react.

But as this is some kind of community "standard" and you see it this way in so much tutorials and examples throughout the web I would also go with this "convention" to lower the entry barrier for new people coming into a project as they see a familiar naming.

I would really like to hear other opinions about this.
@flobacher @sladiri What do you think about it?